md.h

```diff
@@ -191,8 +191,17 @@
             /* https://github.com/ossrs/state-threads/issues/24 */
             #define MD_USE_BUILTIN_SETJMP
             #define MD_GET_SP(_t) *((long *)&((_t)->context[0].__jmpbuf[0]))
-
-        #else
+		
+		#elif defined(__riscv)
+			#if defined(__GLIBC__)   && __GLIBC__ >= 2
```

Comment:
Nice work! 👍

I would like to never use build in setjmp/longjmp, we could always use the `md_linux.S`, because it's simple to maintain only one piece of code.